django/contrib/admin/utils.py

```diff
@@ -481,23 +481,3 @@ def remove_trailing_data_field(fields):
     except NotRelationField:
         fields = fields[:-1]
     return fields
-
-
-def get_limit_choices_to_from_path(model, path):
```

Comment:
Believe it or not, we're importing this one at work. (wasn't me this time :). I'm actually not sure what you've done to replace it.